bin/semantic-release.js

```diff
@@ -41,32 +41,32 @@ var options = _.defaults(
 )
 var plugins = require('../src/lib/plugins')(options)
 
-npmconf.load({}, function (err, conf) {
+npm.load(function (err, npm) {
   if (err) {
     log.error('init', 'Failed to load npm config.', err)
     process.exit(1)
   }
 
-  var npm = {
+  var npmConf = {
     auth: {
       token: env.NPM_TOKEN
     },
-    loglevel: conf.get('loglevel'),
-    registry: require('../src/lib/get-registry')(pkg, conf),
-    tag: (pkg.publishConfig || {}).tag || conf.get('tag') || 'latest'
+    loglevel: npm.config.get('loglevel'),
```

Comment:
Fun fact: Calling `npm.get` will return the config value, but it also console logs the value (which creates a lot of noise on the console). However, `npm.config.get` does not console log the value.